package.json

```diff
@@ -250,6 +250,7 @@
     "chai": "4.1.2",
     "chai-enzyme": "1.0.0-beta.1",
     "check-node-version": "3.2.0",
+    "css-loader": "1.0.0",
```

Comment:
As discussed on Slack, loaders need to go to `dependencies` (rather than `devDependencies`), since our Docker build uses the `--only-production` flag on `npm i`.